utils/utils.go

```diff
@@ -114,3 +116,10 @@ func IsDebugEnabled() (bool, string) {
 		return true, debugValue
 	}
 }
+
+func HandleNoResults(ios *iostreams.IOStreams, msg string) error {
```

Comment:
I think this would fit better in `cmdutil/errors.go`. Also could we call it `NoResultsError`?

Reply:
That was my first instinct, but I felt weird about putting a non-error there xD but yes, good call